docs/src/modules/sandbox/Dependencies.test.js

```diff
@@ -0,0 +1,270 @@
+      'prop-types': 'latest',
+      '@emotion/react': 'latest',
+      '@emotion/styled': 'latest',
+      '@foo-bar/bip': 'latest',
+      '@mui/material': 'latest',
+      '@mui/base': 'latest',
+      '@types/foo-bar__bip': 'latest',
+      '@types/prop-types': 'latest',
+      '@types/react-dom': 'latest',
+      '@types/react': 'latest',
+      typescript: 'latest',
+    });
+  });
+
+  it('should handle multilines', () => {
+    const source = `
+import * as React from 'react';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import {
+  LocalizationProvider as MuiPickersLocalizationProvider,
+  KeyboardTimePicker,
+  KeyboardDatePicker,
+} from '@mui/lab';
+    `;
+
+    const { dependencies } = SandboxDependencies({
+      raw: source,
+      codeVariant: 'JS',
+    });
+
+    expect(dependencies).to.deep.equal({
+      react: 'latest',
+      'react-dom': 'latest',
+      '@emotion/react': 'latest',
+      '@emotion/styled': 'latest',
+      '@mui/material': 'latest',
+      '@mui/lab': 'latest',
+      'date-fns': 'latest',
+    });
+  });
+
+  it('should include core if lab present', () => {
+    const source = `
+import lab from '@mui/lab';
+    `;
+
+    const { dependencies } = SandboxDependencies({
+      raw: source,
+      codeVariant: 'JS',
+    });
+
+    expect(dependencies).to.deep.equal({
+      react: 'latest',
+      'react-dom': 'latest',
+      '@emotion/react': 'latest',
+      '@emotion/styled': 'latest',
+      '@mui/material': 'latest',
+      '@mui/lab': 'latest',
+    });
+  });
+
+  it('can use codesandbox deploys if a commit is given', () => {
+    const source = `
+import * as Material from '@mui/material';
+import * as Base from '@mui/base';
+import * as IconsMaterial from '@mui/icons-material';
+import * as Lab from '@mui/lab';
+import * as Styles from '@mui/styles';
+import * as System from '@mui/system';
+import * as Utils from '@mui/utils';
+    `;
+
+    const { dependencies } = SandboxDependencies(
+      {
+        raw: source,
+        codeVariant: 'JS',
+      },
+      { commitRef: '2d0e8b4daf20b7494c818b6f8c4cc8423bc99d6f' },
+    );
+
+    expect(dependencies).to.deep.equal({
+      react: 'latest',
+      'react-dom': 'latest',
+      '@emotion/react': 'latest',
+      '@emotion/styled': 'latest',
+      '@mui/material': 'https://pkg.csb.dev/mui/material-ui/commit/2d0e8b4d/@mui/material',
+      '@mui/icons-material':
+        'https://pkg.csb.dev/mui/material-ui/commit/2d0e8b4d/@mui/icons-material',
+      '@mui/lab': 'https://pkg.csb.dev/mui/material-ui/commit/2d0e8b4d/@mui/lab',
+      '@mui/styles': 'https://pkg.csb.dev/mui/material-ui/commit/2d0e8b4d/@mui/styles',
+      '@mui/system': 'https://pkg.csb.dev/mui/material-ui/commit/2d0e8b4d/@mui/system',
+      '@mui/utils': 'https://pkg.csb.dev/mui/material-ui/commit/2d0e8b4d/@mui/utils',
+      '@mui/base': 'https://pkg.csb.dev/mui/material-ui/commit/2d0e8b4d/@mui/base',
+    });
+  });
+
+  it('should date adapters', () => {
```

Comment:
Let's also add test for `@mui/x-date-pickers`

Reply:
Nice catch! it is fixed and added a test.